persistence-utils-demo-test/src_test/com/axonivy/utils/persistence/test/IvyTestBase.java

```diff
@@ -38,6 +40,11 @@ public void setup(AppFixture fixture) {
 		// start with system user
 		fixture.loginUser(IvyService.getSystemUser());
 	}
+	
+	@AfterAll
+	public static void down() {
+		IvyEntityManager.getInstance().clearThreadlocals();
```

Comment:
If similar tests as this one will be written customer solution projects, we can also further generalize this tear down with a simple re-usable test-annotation. 

This would mean no inheritance for all tests. Rather just a simple annotation to provide all these features from the base classes. But its a bigger change for this existing test infrastructure. I'd only do it if you think it will be widely used in customer solution projects.